packages/manager/apps/telecom/src/app/telecom/pack/slots/voipLine/activation/translations/Messages_fr_FR.json

```diff
@@ -7,27 +7,27 @@
   "telefony_activation_total_error": "Une erreur est survenue pendant le chargement.",
   "telefony_activation_success_validation": "Votre demande a bien été prise en compte",
   "telefony_activation_an_error_ocurred": "Une erreur est survenue lors de la création de votre ligne",
-  "telefony_activation_shipping_address_label": "Choisissez l’adresse et le mode de livraison de votre téléphone",
-  "telefony_activation_shipping_transporter_label": "Livraison par transporteur (+ {{cost}}€ HT) sous 24h (après envoi)",
-  "telefony_activation_shipping_relay_label": "Livraison gratuite en point Mondial Relay sous 48h (après envoi)",
-  "telefony_activation_shipping_mode": "Mode de livraison:",
-  "telefony_activation_widget_activate_price_free": "Gratuit",
+  "telephony_activation_shipping_address_label": "Choisissez l’adresse et le mode de livraison de votre téléphone",
+  "telephony_activation_shipping_transporter_label": "Livraison par transporteur (+ {{cost}}€ HT) sous 24h (après envoi)",
```

Comment:
We should avoid currencies in the translations as much as possible. Here it is ok because we target only `FR` but let's say we want to target the UK with some French speakers